lisa/transformers/kernel_installer.py

```diff
@@ -93,8 +93,14 @@ def validate(self) -> None:
     def install(self) -> str:
         raise NotImplementedError()
 
+    def get_details(self) -> Dict[str, Any]:
```

Comment:
use a read-only property called `information`. the "details" is not fit here, because the installer doesn't provide "brief". So the information field can be used to provide all information.